docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -35,13 +35,11 @@ This guide requires you to have:
 <Admonition title="Azure AD" type="warning">
 
 Microsoft's Azure Active Directory (Azure AD) offering does not support the
-Kerberos authentication protocol, which is required for Teleport's
-certificate-based authentication.
+Kerberos authentication protocol, which is required the certificate-based 
+authentication described in this section.
 
-At this time, Teleport does not support integration with Azure AD, however
-Teleport Enterprise customers can access Windows desktops (including those
-joined to Azure AD) using local accounts via the process described in [Getting
-Started with Desktop Access](./getting-started.mdx).
+For information about configuring Teleport to work with Azure Active Directory,
+see [Teleport Authentication with Azure Active Directory (AD)](../access-controls/sso/azuread.mdx).
```

Comment:
This should be reverted. There's a difference between using Azure AD as an identity source for users and integrating for access to Windows Desktops. I confirmed with @zmb3 on the Desktop Access team that the current language is still correct.